.github/scripts/install_coherent_freepad.sh

```
@@ -0,0 +1,49 @@
+    if [ "$got" != "$WANT_SHA256" ]; then
+        echo "ERROR: vendored freepad.irx sha256 mismatch (got $got, want $WANT_SHA256)" >&2
+        exit 1
+    fi
+fi
+
+if [ ! -f "$DEST/freepad.irx" ]; then
+    echo "ERROR: stock freepad.irx missing from $DEST (unexpected container layout)" >&2
+    exit 1
+fi
+cp -f "$SRC" "$DEST/freepad.irx"
+echo "== Installed newer-generation freepad.irx ($(wc -c < "$SRC") bytes, sha256 ${WANT_SHA256:0:12}...) -> $DEST/ (#254) =="
```

Shellcheck (0.11.0)

[warning] In POSIX sh, string indexing is undefined.

(SC3057)